### PR TITLE
Pills

### PR DESCRIPTION
**Pull request by s6577t**
- [x] needs a proper chevron
- [x] needs colouring properly
- [x] highlight color for a pill that is the current content (mobile & desktop)
- [x] clickable Contents needs the right pointer

### CODE REVIEW

File: src/components/research/pills.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        font-weight: 600;
+        color: var(--midnight);
+        padding: 12px;
+
+        ${mq(BreakPoint.md)} {
+            display: none;
+            ${withPrefixes`user-select: none;`}
+            padding: 12px 0;
+        }
+    `
```
Comment (rjkerrison) [inline src/components/research/pills.tsx]: this can live outside the `Pills` function

File: src/components/research/pills.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+        ${mq(BreakPoint.md)} {
+            display: block;
+        }
+    `
+
+    return <div css={cssPills}>
+        <div css={cssContent} onClick={() => setOpen(!isOpen)}>Contents<span className="icon"><FontAwesomeIcon icon={icon} /></span></div>
+        <ol css={cssList}>
+            {pills.map((pill, idx) => Pill(pill, idx, [pill.isActive && cssActive, cssPill] ))}
```
Comment (rjkerrison) [inline src/components/research/pills.tsx]: third parameter `[pill.isActive && cssActive, cssPill]` is a little confusing – could we name it as a `const` before we come into this return statement?

File: src/components/research/pills.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    border-top: solid 1px var(--light-grey);
+    display: block;
+
+    ${mq(BreakPoint.md)} {
+        border: none;
+        display: inline-block;
+    }
+`
+
+const Pill = ({ name, href }, key, cssPill) => (
```
Comment (rjkerrison) [inline src/components/research/pills.tsx]: I'd be tempted to make this signature take everything in as properties on the first parameter so we can use `<Pill … />` later

**Review by rjkerrison — APPROVED** [review]
a few suggestions, looks nice

File: src/components/research/pills.tsx [hunk 4]
```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import { css } from "@emotion/core";
+import PropTypes from "prop-types";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faChevronUp, faChevronDown } from "@fortawesome/free-solid-svg-icons";
+import { mq, BreakPoint } from "../../util/mq";
+
+const withPrefixes = (style) => `
```
Comment (tkeburia) [inline src/components/research/pills.tsx:8]: 👍

**Review by tkeburia — APPROVED** [review]

**Pull request merged**